package.json

```diff
@@ -97,6 +104,14 @@
       ],
       "eslint-comments/disable-enable-pair": "off",
       "guard-for-in": "off",
+      "jsdoc/check-values": [
+        "warn",
+        {
+          "allowedLicenses": ["MIT https://github.com/svg-sprite/svg-sprite/blob/main/LICENSE"]
```

Comment:
there is 2 options here
1. keep this
2. change license to "MIT" (see https://spdx.org/licenses/)